spotty/spotty/Data/Models/Episode.swift

```diff
@@ -10,18 +10,26 @@ import Foundation
 struct Episode: Codable, Hashable, BusinessModel {
     
     let description: String
+    let htmlDescription: String
     let id: String
-    let images: [ImageResponse]
+    private let images: [ImageResponse]
```

Comment:
Move properties and functions with private access modifiers below internal/public/open. Look other files also.

Reply:
done